oauth2client/client.py

```diff
@@ -1872,6 +1878,7 @@ def __init__(self, client_id,
         self.device_uri = device_uri
         self.token_info_uri = token_info_uri
         self.authorization_header = authorization_header
+        self.pkce = pkce
```

Comment:
This should be private. If it never needs to be modified or used by anything outside of the class, it should be private.

I'd rather us never expose something than to expose it and later use `DeprecationWarning`.